var/spack/repos/builtin/packages/enzyme/package.py

```diff
@@ -34,7 +35,8 @@ class Enzyme(CMakePackage):
     depends_on("llvm@7:12", when="@0.0.13:0.0.15")
     depends_on("llvm@7:14", when="@0.0.32:0.0.47")
     depends_on("llvm@7:14", when="@0.0.48:0.0.68")
-    depends_on("llvm@9:16", when="@0.0.69:")
+    depends_on("llvm@9:16", when="@0.0.69:0.0.79")
```

Comment:
Did you intend to drop unsupported 10 and 11 from here as well?  Or does that just apply for versions `0.0.80` on?

Reply:
10 and 11 were dropped from 0.0.80 onwards.